challenges/objects-arrays.js

```diff
@@ -80,8 +116,14 @@ zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":
 
 // The zoos need a list of all their animal's names converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
 let lowerCase = [];
+for (let i=0; i<zooAnimals.length; i++) {
```

Comment:
Based on the above instructions, I believe the intent was to use `map()`. 
`let lowerCase = zooAnimals.map((animalName) => {`
`return animalName.animal_name.toLowerCase();`
` });`